camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .detail(ResultErrorBuilder.attribute("enum.values"), result.getEnumChoices(entry.getKey()))
+                        .detail("enum.values", result.getEnumChoices(entry.getKey()))
```

Comment:
for me, upper case on the wire is fine, too. We will soon to start reading binary with HTTP/2 soon anyway ;-P At least for the ipaas-verifier service. Facing to the UI we can make it nicer, but then again, I guess people don't care that much.